### PR TITLE
Add waiting for cke before getting it's content

### PR DESCRIPTION
**Pull request by nikodamn**
- RTE tests: Remove init method
- InfoboxBuilder tests: Remove asserting spinner
- TemplateEditPage: Change implementation of method checking visibility

Reviewers:
@Wikia/west-wing

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/oasis/pages/TemplateEditPage.java [hunk 1]
```diff
@@ -34,7 +36,13 @@ public TemplateClassification getTemplateClassification() {
   }
 
   public boolean isEditAreaDisplayed() {
-    return editArea.isDisplayed();
+    try {
+      wait.forElementVisible(editArea);
+      return editArea.isDisplayed();
+    } catch (NoSuchElementException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/oasis/pages/TemplateEditPage.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by rybmat** [comment]
LGTM

**Pull request merged**